Source/ACE.Server/WorldObjects/Hook.cs

```diff
@@ -27,7 +27,7 @@ public class Hook : Container
 
         public bool HasItem => Inventory != null && Inventory.Count > 0;
 
-        public WorldObject Item => Inventory != null ? Inventory.Values.FirstOrDefault() : null;
+        public WorldObject Item => Inventory?.Values.FirstOrDefault();
```

Comment:
Should not have anything related to null around it.

It is impossible for Inventory to be null